src/RamRelation.h

```diff
@@ -32,29 +32,46 @@ namespace souffle {
 
 /**
  * A RAM Relation in the RAM intermediate representation.
+ * TODO: Make RamRelation a sub-class of RAM node.
```

Comment:
TODOs should be in the form` // TODO (#issueid): text` or `// TODO (userid): text`

Reply:
Done.